src/VlcOptions.h

```diff
@@ -65,6 +65,7 @@ class VlcOptions final {
     string annotateOut() const { return m_annotateOut; }
     bool annotateAll() const { return m_annotateAll; }
     int annotateMin() const { return m_annotateMin; }
+    inline bool countOk(uint64_t count) const { return count >= m_annotateMin; }
```

Comment:
```suggestion
    bool countOk(uint64_t count) const { return count >= m_annotateMin; }
```

Generally inline as a style policy isn't used in C++ class-declared functions as the compiler handles it.